templates/base.html

```diff
@@ -33,7 +33,7 @@
     <!-- social media image -->
     {% block social_media_image %}
       <meta name="twitter:image" content="https://raw.githubusercontent.com/nordic-rse/nordic-rse-materials/main/graphics/www/NordicRSE_TwitterCard.png">
```

Comment:
```suggestion
      <meta name="twitter:image" content="https://raw.githubusercontent.com/nordic-rse/nordic-rse-materials/main/graphics/www/NRSE_some.png">
```

to avoid things going out of sync. If we have officially abandonded twitter, coul also just remove this line